.github/build-packages.php

```diff
@@ -44,6 +44,18 @@
         $key = isset($packageData['require']['symfony/stimulus-bundle']) ? 'require' : 'require-dev';
         $packageData[$key]['symfony/stimulus-bundle'] = '@dev';
     }
+    
+    if (isset($packageData['require']['symfony/ux-map'])
+        || isset($packageData['require-dev']['symfony/ux-map'])
+    ) {
+        $repositories[] = [
+            'type' => 'path',
+            'url' => '../../../',
```

Comment:
This will only work for map bridges, right ?

Reply:
Yes, for the moment only the Map bridges require symfony/ux-map (as dev deps)